src/main/java/rest_api/petshop/OrderStatus.java

```diff
@@ -0,0 +1,5 @@
+package rest_api.petshop;
+
+public enum OrderStatus {
+    placed, approved, delivered
```

Comment:
Please, follow the Java Code Convention and keep the Enums naming format

Reply:
Done